### ISSUE TITLE
Hide "Renew" and "Records" actions inside ENS token

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-android#2015]
They are not working, so it's better if we hide these functionalities for now, till we have this fixed one day. So please make just one button, "Transfer". 

Stays like this for 15 minutes already and no change.
![2021-09-13 09 23 17](https://user-images.githubusercontent.com/51817359/133112127-69adc170-aeb9-43d0-87c0-7b89a9461f77.jpg)

-----
Temporary, new design
<img width="772" alt="Screenshot 2021-09-13 at 09 28 59" src="https://user-images.githubusercontent.com/51817359/133112407-0b7f0d2c-b3d7-4ce9-ab4f-46e681d13a85.png">

@hboon I think this is what we should do with all not-working features. Hide them for now.

**Comment by hboon** [comment]
@colourfreak yes. If they don’t work or not work well enough, we should hide or remove them.

**Comment by JamesSmartCell** [comment]
This is a TokenScript issue - does the script now fail?

**Comment by mpaschenko** [comment]
@JamesSmartCell so what to do with that - just add check for the **Renew** option and disable that by hardcode?

**Comment by JamesSmartCell** [comment]
I will have a look at the script see if it's easy to update. I have to renew my ENS soon anyway, and had a small involvement with the design of this script.

**Comment by colourfreak** [comment]
> I will have a look at the script see if it's easy to update. I have to renew my ENS soon anyway, and had a small involvement with the design of this script.

Ideally with activity related to these transactions. I mean, let's aim for the best possible feature. So for example when I set ENS avatar, it was unclear looking at Activity what I have just did. 

Example:
https://github.com/AlphaWallet/alpha-wallet-ios/issues/3278#issuecomment-940931777

**Comment by JamesSmartCell** [comment]
> Example: [AlphaWallet/alpha-wallet-ios#3278 (comment)](https://github.com/AlphaWallet/alpha-wallet-ios/issues/3278#issuecomment-940931777)

I like this flow much better.

FWIW, I think that activity page you currently get to when you click the activityview item was primarily intended for TokenScript enabled transactions that have the expanded activity text / graphics. I'll post an example when I can find one.

But, it doesn't need to be there for anything other than a TokenScript transaction that has an expanded view.
